### PR TITLE
Correcting SectionLabel margins below 1008px to match designs

### LINKED ISSUE #5296: Spacing between last body text and section divider for related content is too large below 1008px
Jira [WSMEDIA-145](https://jira.dev.bbc.co.uk/browse/WSMEDIA-145)

**Describe the bug**
Spacing between last paragraph / text in the copy body and the section divider for related content should be:

- < 1008px 48px
- .>= 1008px 56px 

Also see designs: 

- < 1008 [zpl.io/blAYmAe](http://zpl.io/blAYmAe)
- .>= 1008 [zpl.io/VYnEALk](http://zpl.io/VYnEALk)

**To Reproduce**
1. Go to https://www.bbc.com/pidgin/50676196
2. Note that there is too much spacing between that last line of text in the story and the related content ('Another thing we de for inside dis tori') section above and below 1008px. 

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves #5296

**Overall change:** 
The total margin below article content and the recommended content block is now 56px above 1008px screen width and 48px below. The [issue](https://github.com/bbc/simorgh/issues/5296) describes the current problem with the margins.

**Code changes:**
To avoid changing the Psammead component itself, `SectionLabel` was wrapped in a styled component within `src/app/containers/CpsRelatedContent` and its margin-top property set to 0px. The `StyledGhostGrid` component within `src/app/pages/CpsMap` `.../CpsPgl` `.../CpsSty` has had its padding-bottom changed below 1008px to 16px and 24px above. 

These changes give a total distance between the section divider and the final piece of content of 48px below 1008px and 56px above.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)

### CODE REVIEW

File: src/app/containers/CpsRelatedContent/index.jsx [hunk 1]
```diff
@@ -28,6 +28,10 @@ const Wrapper = styled(GridItemConstrainedLarge)`
   z-index: 0;
 `;
 
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0rem;
```
Comment (ryanmccombe) [inline src/app/containers/CpsRelatedContent/index.jsx]:
```suggestion
  margin-top: 0;
```

**Review by ryanmccombe — APPROVED** [review]
Looks good to me, assuming the stories pod are okay with it.

One thing that stands out is the duplication around the different CPS page types - I think it's okay to the extent in this PR, but does seem like it is going to be an ongoing requirement where these page types share logic like this

Breaking out `CPSGhostGrid` and `CPSTimestamp` into external components might be a bit much, but it may be worth checking with the story pod about this - it seems like it's going to be a common problem

**Review by karinathomasbbc — APPROVED** [review]
Happy to approve this, although I would reach out to the Stories team to check if they are OK for the changes to be made on their pages too.

**Review by simonsinclair — APPROVED** [review]
This looks good, @RichardPK. It's helpful to have changes like this consolidated - thanks for that.

**Pull request merged**